src/main/java/ulcrs/models/session/Session.java

```diff
@@ -0,0 +1,52 @@
+package ulcrs.models.session;
+
+import ulcrs.models.schedule.Schedule;
+
+public class Session {
+
+    private String name;
+    private Schedule existingSchedule;
+    private long generatedTimestamp;
```

Comment:
Use full DateTime objects or whatever Java's or JodaTime's equivalent is.

Reply:
Ahh yeah good catch, will update.

Reply:
Updated to LocalTime, not a date, as Jason pointed out in lecture. Will update this again.